src/rust/bitbox02-rust/src/workflow/menu.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::bb02_async::option;
+use bitbox02::ui;
+use core::cell::RefCell;
+
+extern crate alloc;
+use alloc::boxed::Box;
+
+pub async fn menu() {
+    let result = RefCell::new(None);
+    let mut component = ui::menu_create(ui::MenuParams {
```

Comment:
When this demo menu is created, there's no "Continue" top right corner. I tap the `>` arrow bottom right and "Continue" appears. No matter if I tap `<` or `>` arrows, the "Continue" is always present now.

I can also tap "Select" bottom middle and the `select_word_cb` gets called without tapping "Continue". Doesn't feel exactly right but I guess it's the underlying implementation in C, nothing to do with this PR.

Reply:
Yeah this is intended. The two uses cases are: scroll through mnemonics, where Continue appears once you are through. There is no `Select` in this.

The other is confirming a mnemonic word, where there is `Select`, but no Continue.